### PR TITLE
Top Bar: Component + Story + Snapshot

### PR DESCRIPTION
**Pull request by Mar0cas**

### CODE REVIEW

File: packages/organisms/top-bar/src/TopBar.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import Html, { ContentProps } from '@creat/html';
+import Section from '@creat/section';
+import React from 'react';
+import { Defaults } from './index';
+
+const TopBar = ({ title, url, ...props }: any) => {
```
Comment (rebecatalkdesk) [inline packages/organisms/top-bar/src/TopBar.tsx:6]: `{ title, url, ...props }: Props`

File: packages/organisms/top-bar/src/__stories__/TopBar.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import TopBar from "../TopBar";
+import mockJson from "./mock.json";
+
+export default {
+  title: "Organisms/TopBar",
+};
+
+export const topBar = (props: any) => {
```
Comment (rebecatalkdesk) [inline packages/organisms/top-bar/src/__stories__/TopBar.stories.tsx:9]: `props: Props`

File: packages/organisms/top-bar/src/__stories__/mock.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "acf_fc_layout": "top_bar",
+  "title": "Augment your on-premises ACD with Talkdesk Boost: <u>Learn More</u>",
+  "url": "https://www.talkdesk.com/call-center-software/platform/boost/",
+  "color": "teal",
+  "section": ["-stacked"],
+  "type": "full big",
```
Comment (rebecatalkdesk) [inline packages/organisms/top-bar/src/__stories__/mock.json:7]: Is there a type full(space)big?

File: packages/organisms/top-bar/src/component/Types.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { SectionProps } from '@creat/section';
+import { ReactNode } from 'react';
+
+interface Props extends SectionProps {
+  children?: ReactNode;
```
Comment (rebecatalkdesk) [inline packages/organisms/top-bar/src/component/Types.ts:5]: Should be removed for two reasons, it's not used in component code and SectionProps already has a children prop.

File: packages/organisms/top-bar/src/__stories__/mock.json (hunk 3 above)
Comment (Mar0cas) [reply packages/organisms/top-bar/src/__stories__/mock.json:7]: Yes, its what come in the schema

**Pull request merged**